jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java

```diff
@@ -34,6 +34,7 @@
 /** Obtains information about a {@link MavenProject}. */
 class MavenProjectProperties implements ProjectProperties {
 
+  public static final String TOOL_NAME = "jib-maven-plugin";
```

Comment:
I think it'd be good to add some javadocs for these constants now that there's 3 so it's immediately clear what each's purpose is.